packages/circuits/circuits/identity.circom

```diff
@@ -3,10 +3,10 @@ pragma circom 2.1.0;
 include "./circomlib/circuits/poseidon.circom";
 
 template IdentityCommitment() {
-  signal input secret;
```

Comment:
This was named secret to avoid duplicate text in circuits.

e.g.

```
template identity_commitment;
identity_commitment.secret <== value;
// instead of
identity_commitment.identity_secret <== value;
```

Reply:
I am trying to align the names in all circuits
so it won't confuse developers when they use different circuits with different input signals